src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -160,7 +158,7 @@ export async function buildMapView(id: string, dotNetReference: any, long: numbe
                 });
                 let center;
                 let spatialRef;
-                if (spatialReference !== undefined && spatialReference !== null) {
+                if (hasValue(spatialReference)) {
```

Comment:
Nice!

Reply:
you can only write the same thing so many times.... 🙂